util/gvalid/gvalid_validator.go

```diff
@@ -28,8 +28,10 @@ type Validator struct {
 	ruleFuncMap                       map[string]RuleFunc // ruleFuncMap stores custom rule functions for current Validator.
 	useAssocInsteadOfObjectAttributes bool                // Using `assoc` as its validation source instead of attribute values from `Object`.
 	bail                              bool                // Stop validation after the first validation error.
+	fieldBail                         bool                // Stop field validation after the first validation error.
```

Comment:
The attribute `fieldBail` might be duplicated with attribute `bail`.

Reply:
The attribute `fieldBail` is equivalent to adding bail to each field of the struct